#PCM outliar rejection params
multi_robot_pcm_queue_size: 5
multi_robot_min_pcm: 2

#noise models for multo-robot factors
inter_robot_sigmas: [0.1, 0.1, 0.01]
partner_robot_sigmas: [0.1, 0.1, 0.01] #[0.25, 0.25, 0.0872665]

#when to resend the trajectory
resend_translation: .1 #if this is -1 then the trajectory is NOT resent 
resend_rotation: 3.0

#res for the point cloud compression module, if you want this on make sure to say so below in study
point_compression_resolution: .25

#ablation study params
study:
  case: 7
  overlap: True
  count: True
  ratio: True
  context: True
  pcm: True
  point_compression: False

#registration params
mrr: #(multi-robot-registration)
  min_points: 75
  points_ratio: 2.0
  min_overlap: .55
  sampling_points: 500
  iterations: 5
  tolerance: 0.01
  max_translation: 10.0 #meters
  max_rotation: 0.785398 #radians
  k_neighbors: 5 #3
  max_scan_context: 140
  max_tree_cost: 20

#scan context params
sc:
  number_of_scans: 1
  bearing_bins: 100
  max_bearing: 180
  range_bins: 16
  max_range: 40 

#loop closure search params for partner robot
#after global regitration
plcs: #(partner loop closure search)
  max_translation_search: 10 #meters
  max_rotation_search: 1.0472 #radians
